apps/judicial-system/web/src/routes/Prosecutor/Indictments/CaseFile/CaseFile.tsx

```diff
@@ -65,7 +65,8 @@ const CaseFile = () => {
                     workingCase.caseFiles?.filter(
                       (caseFile) =>
                         caseFile.policeCaseNumber === policeCaseNumber &&
-                        caseFile.category === CaseFileCategory.CASE_FILE,
+                        (caseFile.category === CaseFileCategory.CASE_FILE ||
+                          caseFile.category === CaseFileCategory.INVOICE),
```

Comment:
Invoices should not be included in case files records. They are standalone case files listed under "other case files" in the client.